### ISSUE TITLE
Zeebe self-managed logging configuration is broken

### ISSUE THREAD

**Issue by npepinpe** [camunda/camunda-platform-docs#1078]
You can see [here](https://docs.camunda.io/docs/self-managed/zeebe-deployment/configuration/#google-stackdriver-json-logging) that we try to "include" a file from somewhere. I'm not sure if this is meant to work, or if this is just a remnant from our previous docs (probably the latter).

We should either remove this block of code, or fix it (is it possible? do we even have a way to embed code from other sources?).

**Comment by akeller** [comment]
Quick fix - let's remove it.

Slightly longer fix - link to https://github.com/camunda/zeebe/blob/main/dist/src/main/config/log4j2.xml. Since this file looks relatively static (no updates since May 2021), and relatively small, we could copy the whole thing into a code block. 

Possible more involved - stick it in an iframe or something MDX compatible(?). I would punt this to @pepopowitz to work out the details.

**Comment by npepinpe** [comment]
I'm fine with removing it. It would be nice long term to include snippets of code directly from the main repo, esp. for configuration and code examples. I did that in another project for sample code - the sample code was then also tested as part of the project CI, so we were sure examples would always work.

Anyway, for this one I think it's fine to remove for now :+1:

**Comment by Zelldon** [comment]
Look what I found https://github.com/saucelabs/docusaurus-theme-github-codeblock :D 

Sounds like what we would want to have just writing:

```
  ```js reference title="Example"
https://github.com/camunda/zeebe/blob/main/dist/src/main/config/log4j2.xml
\`\`\`
```

**Comment by menski** [comment]
fixed by #1106

**Comment by akeller** [comment]
> https://github.com/saucelabs/docusaurus-theme-github-codeblock

@pepopowitz good backlog item & partnering with @christinaausley to find where we could use it today, then make it available to our engineers for future docs. 

In fact, I see this helping to solve our accessibility issue with BPMN diagrams by embedding the XML. Maybe as a toggle or tabbed experience? 🤔 I'll create a new issue here.
